docs/project/evolution.md

```diff
@@ -314,9 +314,17 @@ The author of a proposal is not required to include changes to the design
 documentation as part of a proposal, and it may in some cases be preferable to
 decouple the proposal process from updating the design. When accepted, the
 proposal would then be implemented through a series of future PRs to the rest of
-the project, and the proposal document should describe what is being proposed in
-enough detail to validate that those future PRs properly implement the proposed
-direction.
+the project. If the proposal PR defers any documentation changes in this way,
+the proposal document should describe what is being proposed in enough detail to
+validate that those future PRs properly implement the proposed direction, and
+the proposal PR should add "TODO" comments in the places where significant
+future changes will be needed (including adding new placeholder documents if
+needed), with links back to the proposal document. These comments should be kept
+close to the passages that need to be changed. This is intended to ensure that
```

Comment:
Perhaps it'd make sense to just have one TODO in a document where several passages need to be changed? For example, #5168 has several TODOs in details.md: perhaps one at the top should be enough, rather than one per passage?

Reply:
I assume you mean #4230? If so, I don't follow -- it's adding a new design document `declaring_entities.md`, not modifying any existing documents, so where would TODOs have gone, other than in an empty placeholder version of `declaring_entities.md`?

Reply:
> I assume you mean #4230? If so, I don't follow -- it's adding a new design document `declaring_entities.md`, not modifying any existing documents, so where would TODOs have gone, other than in an empty placeholder version of `declaring_entities.md`?

Yes, #4230, sorry.

Yes, I think that's my point -- except maybe individually, people wouldn't have thought to add a new file, and instead we'd have TODOs sprinkled around potentially-relevant passages.